### DISCUSSION TITLE
is it possible to use next-auth + firebase auth without using firestore?

### DISCUSSION THREAD

**Discussion by eddyhdzg-solarx** [nextauthjs/next-auth#8849] [Help]
### Summary

I want to use firebase authentication with their multiple providers cause it's easy to use, and can be shared both in react and react-native.

![image](https://github.com/nextauthjs/next-auth/assets/111922363/c6bb8968-a402-41d3-9225-f4b4344306ef)

**Comment by TheIrvingBarajas** [comment]
@eddyhdzg-solarx This is what I would like to do as well. Curious if you ended up with an alternative solution?

**Comment by ccrowley96** [comment]
Also curious if this is feasible!
